content/en/docs/09/job_control.md

```diff
@@ -72,3 +74,42 @@ Updated `.gitlab-ci.yml` file for this lab:
 {{< highlight yaml "hl_lines=6 62-68" >}}{{< readfile file="manifests/09.0/.gitlab-ci.yml" >}}{{< /highlight >}}
 
 {{% /details %}}
+
+
+## {{% param sectionnumber %}}.5: Rules
+
+Available since GitLab 12.3.
+
+Rules are evaluated in the specified order - until the first one matches. Depending on the configuration, the job is then either in- or excluded from the pipeline.
+
+`rules` accepts an array of rules defined with:
+
+* `if`
+* `changes`
+* `exists`
+* `allow_failure`
+* `variables`
+* `when`
+
+**Important**: `rules` replaces `only/except`. Thus they can not be used together in the same job. If you configure a job using both keywords, GitLab will return a `key may not be used with rules` error!
+
+
+## Task {{% param sectionnumber %}}.6: Job control using rules
+
+* Implement the same rule using `rules` instead.
+
+Commit and push these changes to the `release` branch. It should behave exactly the same as in the previous excercise.
+
+
+## Solution
+
+```yaml
```

Comment:
Include the solution from file located inside the manifest folder. Highlight the changes.
Like lab 5 you can have multiple solution files.